### PR TITLE
vmagent: add minimal scrape file exampe for vmagent quick start

### PR DESCRIPTION
**Pull request by tenmozes**
Signed-off-by: Artem Navoiev <tenmozes@gmail.com>

### CODE REVIEW

File: app/vmagent/README.md [hunk 1]
```diff
@@ -60,6 +60,17 @@ Example command line:
 /path/to/vmagent -promscrape.config=/path/to/prometheus.yml -remoteWrite.url=https://victoria-metrics-host:8428/api/v1/write
 ```
 
+Example of minimal scrape configuration for `-promscrape.config` argument:
```
Comment (hagen1778) [inline app/vmagent/README.md:63]: Have you considered pointing to https://github.com/VictoriaMetrics/VictoriaMetrics/blob/master/deployment/docker/prometheus.yml instead?

Comment (tenmozes) [reply app/vmagent/README.md:63]: sounds good let me change

**Review by f41gh7 — APPROVED** [review]
LGTM

**Comment by f41gh7** [comment]
Thanks for contribution!

**Pull request merged**